airbyte-cdk/python/airbyte_cdk/utils/airbyte_secrets_utils.py

```diff
@@ -18,3 +18,19 @@ def get_secrets(source: Source, config: Mapping[str, Any], logger: logging.Logge
         ".".join(key.split(".")[:1]) for key, value in flattened_key_values.items() if value and key.endswith("airbyte_secret")
     ]
     return [str(get_value_by_dot_notation(config, key)) for key in secret_key_names if config.get(key)]
+
+
+class AirbyteSecretHelper:
```

Comment:
Arguably this is not very pythonic in that we're defining a class with the intention to never instantiate it, and using classmethods to update and use a class attribute. This sort of global attribute storage / updating / usage could be done at the module level, e.g. at top level of this .py
```
__SECRETS_FROM_CONFIG = []

def update_secrets(secrets: List[str]):
        global __SECRETS_FROM_CONFIG
        __SECRETS_FROM_CONFIG = secrets

def filter_secrets(string: str) -> str:
        global __SECRETS_FROM_CONFIG
        for secret in __SECRETS_FROM_CONFIG:
            string = string.replace(secret, "****")
        return string
```
then elsewhere to use
```
from airbyte_secret_utils import update_secrets, filter_secrets
# and use the functions directly
```

This achieves exactly the same thing but more pythonic imo.

Reply:
Interesting! Thanks for the tip, changed in [94956f2](https://github.com/airbytehq/airbyte/pull/12593/commits/94956f25e592110f1ec48a5b84f3414d4170f963)